packages/bus-query-operation/lib/Bindings.ts

```diff
@@ -59,12 +60,14 @@ export function materializeOperation(
   options: {
     strictTargetVariables?: boolean;
     bindFilter?: boolean;
+    originalBindings?: Bindings;
   } = {},
-  originalBindings?: Bindings,
+  
 ): Algebra.Operation {
   options = {
     strictTargetVariables: 'strictTargetVariables' in options ? options.strictTargetVariables : false,
     bindFilter: 'bindFilter' in options ? options.bindFilter : true,
+    originalBindings: 'originalBindings' in options ? options.originalBindings : bindings,
```

Comment:
This line should make the line I commented on a bit lower redundant, but I left the second line in because the type checker thinks options.originalBindings can be undefined.
That seemed better than using @ts ignore everywhere. If there's a better options I'd be glad to hear it!